helm/daq-config-server/values.yaml

```diff
@@ -48,9 +48,23 @@ readinessProbe:
     path: /healthz
     port: http
 
-volumes: []
+# Send logs to the dodal graylog stream for now until a more suitable stream exists
+logging_config:
```

Comment:
Should: I would nest this under config before you inevitably need more configs later on